input/fsh/aliases.fsh

```diff
@@ -1,7 +1,14 @@
+// ---- Systems
+
 Alias: $v2-0203 = http://interopsante.org/CodeSystem/v2-0203
 Alias: $JDV-J165-TypeCreneau-SAS = https://mos.esante.gouv.fr/NOS/JDV_J165-TypeCreneau-SAS/FHIR/JDV-J165-TypeCreneau-SAS
 Alias: $v3-ActEncounterCode = http://terminology.hl7.org/ValueSet/v3-ActEncounterCode
 Alias: $fr-v2-0203 = http://interopsante.org/fhir/CodeSystem/fr-v2-0203
 Alias: $v3-ActCode = http://terminology.hl7.org/CodeSystem/v3-ActCode
 Alias: $fr-location-identifier-type = http://interopsante.org/fhir/CodeSystem/fr-location-identifier-type
```

Comment:
http://interopsante.org/fhir/CodeSystem/fr-location-identifier-type
=> 
<img width="236" alt="image" src="https://github.com/ansforge/IG-fhir-service-acces-aux-soins/assets/117643165/e634be66-2389-4df5-a58a-4126fd556818">

Reply:
ValueSet bien présent dans FrCore 1.1.0 mais pas résolvable : https://simplifier.net/packages/hl7.fhir.fr.core/1.1.0/files/783642

Reply:
Peut être parce que le numéro de version du ValueSet est erroné :

<img width="644" alt="image" src="https://github.com/ansforge/IG-fhir-service-acces-aux-soins/assets/48218773/943ccbb4-b764-4a7e-a937-a1ebc3745ba9">

Reply:
L'erreur ValueSet not found vient peut être du numéro de version qui est erroné :

<img width="644" alt="image" src="https://github.com/ansforge/IG-fhir-service-acces-aux-soins/assets/48218773/943ccbb4-b764-4a7e-a937-a1ebc3745ba9">

Reply:
Je ne pense pas, c'est pareil pour les autres code system de ce package 1.1.0 on dirait. Ou alors on aurait cette erreur en utilisant tous les code système de ce package du coup ?